Microsoft-Graph-ASPNET-Excel-Donations/Web.config

```diff
@@ -13,8 +13,8 @@
     <!-- to save time in the quick start labs. -->
     <!-- This client ID and secret will be invalidated after //build. -->
     <!-- In normal circumstances, you should never share client IDs and secrets -->
-    <add key="ida:AppId" value="e684617c-2d2e-4182-ad19-2d996a46d00a" />
-    <add key="ida:AppSecret" value="fZ[f|Cf&gt;!FC#[!/%Z@u_&gt;PThtam[W[-b]&gt;4uarW*;W@&amp;bn#)0!_)/0Uw6%yt=i+" />
+    <add key="ida:AppId" value="ENTER_YOUR_CLIENT_ID" />
+    <add key="ida:AppSecret" value="*ENTER_YOUR_SECRET" />
```

Comment:
Really good catch here to remove the AppId and AppSecret! 
Clean up the value of the *ida:AppSecret* to remove the redundant * before the value.